devicon.json

```diff
@@ -3396,6 +3780,9 @@
     },
     {
         "name": "maya",
+        "altnames": [
+            "md"
```

Comment:
`md` is already used for `markdown`. Does it realy apply to `maya`?

Reply:
Nope, another mistake.
Fixed here: 6e9a895871f9c489551e347f4b9c05ee0830f8ae